web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -121,36 +121,6 @@ describe("<SidebarCardsContainer />", () => {
       const bottomCardSection = screen.getByTestId("bottom-cards");
       expect(within(bottomCardSection).getByTestId("fake-below-card")).toBeInTheDocument();
     });
-
-    it("removes successful registration card when it's closed", async () => {
```

Comment:
[boulder] Same as my comment regarding the welcome card story last week, this test should not be removed. It is not testing the success card, it is testing the "close button" functionality and just happens to use the success card as its test subject. Without this test, there is no test for the behavior that happens when a card has `hasCloseButton` = true and what happens when that close button is clicked. 

This test should continue to exist, it should just be a made-up card instead of the success card.